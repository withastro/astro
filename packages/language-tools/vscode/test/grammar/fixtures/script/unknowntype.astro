<script type="unknown">
	"Hello"
</script>
