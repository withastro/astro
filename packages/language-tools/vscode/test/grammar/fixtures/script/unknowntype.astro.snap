><script type="unknown">
#^ source.astro meta.scope.tag.script.astro meta.script.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^^^^ source.astro meta.scope.tag.script.astro meta.script.astro meta.tag.start.astro entity.name.tag.astro
#       ^ source.astro meta.scope.tag.script.astro meta.script.astro meta.lang.unknown.astro meta.tag.start.astro
#        ^^^^ source.astro meta.scope.tag.script.astro meta.script.astro meta.lang.unknown.astro meta.tag.start.astro meta.attribute.type.astro entity.other.attribute-name.astro
#            ^ source.astro meta.scope.tag.script.astro meta.script.astro meta.lang.unknown.astro meta.tag.start.astro meta.attribute.type.astro punctuation.separator.key-value.astro
#             ^ source.astro meta.scope.tag.script.astro meta.script.astro meta.lang.unknown.astro meta.tag.start.astro meta.attribute.type.astro string.quoted.astro punctuation.definition.string.begin.astro
#              ^^^^^^^ source.astro meta.scope.tag.script.astro meta.script.astro meta.lang.unknown.astro meta.tag.start.astro meta.attribute.type.astro string.quoted.astro
#                     ^ source.astro meta.scope.tag.script.astro meta.script.astro meta.lang.unknown.astro meta.tag.start.astro meta.attribute.type.astro string.quoted.astro punctuation.definition.string.end.astro
#                      ^ source.astro meta.scope.tag.script.astro meta.script.astro meta.lang.unknown.astro meta.tag.start.astro punctuation.definition.tag.end.astro
>	"Hello"
#^^^^^^^^^ source.astro meta.scope.tag.script.astro meta.script.astro meta.lang.unknown.astro
></script>
#^^ source.astro meta.scope.tag.script.astro meta.script.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#  ^^^^^^ source.astro meta.scope.tag.script.astro meta.script.astro meta.tag.end.astro entity.name.tag.astro
#        ^ source.astro meta.scope.tag.script.astro meta.script.astro meta.tag.end.astro punctuation.definition.tag.end.astro
>